wiki/en/Installation-for-Linux.md

```diff
@@ -54,24 +54,45 @@ For installers on other distributions, see their package managers and [Repology]
 
 ## Set up your hardware
 
+The Jamulus client needs to connect to a running [JACK](https://jackaudio.org/) server in order to start. You need to find out which is the low-latency sound server of your distribution.
+If your distribution uses JACK, check out how to [configure JACK with QjackCtl](Installation-for-Linux#configure-jack-with-qjackctl), on the other hand, if it is PipeWire, check out the section called [configure PipeWire](Installation-for-Linux#configure-pipewire) instead.
+
 ### Configure JACK with QjackCtl
 
-Jamulus Clients need [JACK](https://jackaudio.org/) to run, but you need to configure that first. The recommended method is to use `QjackCtl`.
+To run a [JACK](https://jackaudio.org/) server, the Jamulus team's recommended method is to use `QjackCtl`.
 
-1. Launch QjackCtl. You will see the **Qt JACK Control utility main page**
+1. Launch QjackCtl. You will see the **Qt JACK Control utility main page**.
 2. Configure your audio hardware as follows (the exact settings for JACK will depend on what your audio hardware is capable of):
 
-- Set the audio **Interface** to the one you want (there may be several in the list)
-- Set the **Sample Rate to 48000**
-- Set the **Frames/Period to 128** and Periods/Buffer at 2 at first
+- Set the audio **Interface** to the one you want (there may be several in the list).
+- Set the **Sample Rate to 48000**.
+- Set the **Frames/Period to 128** and **Periods/Buffer at 2** at first.
+
+Restart JACK for the new settings to take effect and rendez-vous at [start Jamulus](Installation-for-Linux#start-jamulus) section.
 
-Restart JACK for the new settings to take effect
+### Configure PipeWire
+
+PipeWire provides its own JACK server. When you launch the Jamulus Client, PipeWire automatically runs a JACK server. However, there are a few parameters to adjust.
+You need to set your audio interface to the "pro-audio" profile and define PipeWire's "rate" and "quantum" parameters beforehand.
+More informations about the configuration in the [PipeWire's wiki](https://gitlab.freedesktop.org/pipewire/pipewire/-/wikis/Config-JACK#jack-server).
+
+Follow these steps to set up your system:
+1. Open `pavucontrol` (the audio mixer) and choose the "pro-audio" profile for your audio interface in the configuration tab.
+2. In a terminal use the two following commands to force PipeWire's rate and quantum:
+```bash
+pw-metadata -n settings 0 clock.force-rate 48000
+pw-metadata -n settings 0 clock.force-quantum 128
+```
+After that, you can continue reading the rest of this documentation. Your system is ready.
 
 ### Start Jamulus
 
-With JACK running and configured, launch Jamulus.
+With JACK or PipeWire configured, launch Jamulus.
+
+If you get problems with sound breaking up (in particular XRUN errors reported by JACK/QjackCtl) try setting bigger values (e.g. 256 frames or 3 periods).
```

Comment:
If this _only_ applies to `jackd`, that should be made clear.  Otherwise, it needs explaining in a way that applies to both `jackd` and PipeWire.

Reply:
I would prefer to remove this sentence.

Reply:
Done.